README.md

```diff
@@ -1410,6 +1410,7 @@ Table of Contents
    * [Eclipse Che](https://www.eclipse.org/che/) - Web based and Kubernetes-Native IDE for Developer Teams with multi-language support. Open Source and community driven. A online instance hosted by Red Hat is available at [workspaces.openshift.com](https://workspaces.openshift.com/).
    * [fakejson.com](https://fakejson.com/) — FakeJSON helps you quickly generate fake data using its API. Make an API request describing what you want and how you want it. The API returns it all in JSON. Speed up the go to market process for ideas and fake it till you make it.
    * [gitpod.io](https://www.gitpod.io) — Instant, ready-to-code dev environments for GitHub projects. Free for open source.
+   * [HasteBin](https://hastebin.com/) - A simple web based "bin" to share/store your code, snippet, log, or any text file. Like PasteBin, Github Gist but simpler. Basically free to use. Also support API. Open source, in case you wish to self-host.
```

Comment:
I am not sure I want to list pastebins here there are just too many and they are just minimally useful. We probably have some already but not adding more